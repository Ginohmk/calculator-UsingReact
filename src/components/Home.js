import React from 'react';

export default function Home() {
  return (
    <div className="main" data-testid="main-container">
      <h2>Welcome to our page!</h2>
      <p>
        lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem
        ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem
        ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem
        ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem
        ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem
        ipsumcontel lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem
        ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem
        ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem
        ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem
        ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem
        ipsumcontel
      </p>
      <p>
        lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem
        ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem
        ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem
        ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem
        ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem
        ipsumcontel lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem
        ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem
        ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem
        ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem
        ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem
        ipsumcontel
      </p>
    </div>
  );
}
