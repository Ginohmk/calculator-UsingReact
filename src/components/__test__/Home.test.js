import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

describe('TEst for Home component', () => {
  test('Home snapShot test', () => {
    const tree = renderer.create(<Home />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
