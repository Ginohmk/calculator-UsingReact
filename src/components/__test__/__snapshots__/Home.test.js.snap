// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEst for Home component Home snapShot test 1`] = `
<div
  className="main"
>
  <h2>
    Welcome to our page!
  </h2>
  <p>
    lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel
  </p>
  <p>
    lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontel lorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel lorem ipsumcontellorem ipsumcontel
  </p>
</div>
`;
